components/cqrs/event_processor_group.go

```diff
@@ -248,10 +248,10 @@ func (p EventGroupProcessor) routerHandlerGroupFunc(handlers []GroupEventHandler
 				logger.Debug("Error when handling event", watermill.LogFields{"err": err})
 				return err
 			}
-
-			return nil
 		}
 
+		return nil
```

Comment:
The `return` statement here skips the `if` below entirely, right? Is that on purpose?

Reply:
indeed, this would be a problem, it's fixed here: https://github.com/ThreeDotsLabs/watermill/commit/9d5e2da133397fcfb6e8f00eb424df91005e9c7a